integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/End2EndTestITStep.java

```diff
@@ -251,4 +254,16 @@ public void testMetrics() {
         assertThat(ingressMetrics).contains("http_server_requests_seconds_count{method=\"POST\",outcome=\"SUCCESS\",status=\"200\",uri=\"/events/plain\",} 1.0");
     }
 
+    @After
+    public void cleanUp() {
+        if (IntStream.range(0, getBridgeList().getItems().size()).anyMatch(b -> getBridgeList().getItems().get(b).getId().equals(bridgeId))) {
+            if (listProcessors(bridgeId).getSize() > 0) {
+                IntStream.range(0, listProcessors(bridgeId).getItems().size()).forEach(p -> deleteProcessor(bridgeId, listProcessors(bridgeId).getItems().get(p).getId()));
+                Awaitility.await().atMost(Duration.ofMinutes(2)).pollInterval(Duration.ofSeconds(5)).until(() -> listProcessors(bridgeId).getSize() == 0);
+                deleteBridge(bridgeId);
```

Comment:
Not necessary as the bridge is deleted anyway below.

Reply:
No here deleting bridge in case of processor associated ..In else case bridge will get deleted if processor not associated

Reply:
Yes sorry confused..i will remove this method here ..anyway bridge will be deleted below